REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.6
+julia 1.0
```

Comment:
WHY?

Reply:
I thought that the compatibility issue with 1.0 reported by Travis is related to this line. But it is not.

Reply:
Okay, apparently I had not installed GMP here. Now it works.

The build still fails, now with a different case:
```
ERROR: Unsatisfiable requirements detected for package JuMP
```

Reply:
To me this sounds like an external `REQUIRE` battle between [`ReverseDiffSparse`](https://github.com/mlubin/ReverseDiffSparse.jl), [`ForwardDiff`](https://github.com/JuliaDiff/ForwardDiff.jl), [`JuMP`](https://github.com/JuliaOpt/JuMP.jl), and [`Polyhedra`](https://github.com/JuliaPolyhedra/Polyhedra.jl). If that is true, we cannot win here (for now).

Reply:
[This](https://discourse.julialang.org/t/package-manager-what-are-explicit-requirements/18616/2) looks like a similar problem, if not the same one..